ios/PurchasesHybridCommon/PurchasesHybridCommon/NSDate+HybridAdditions.swift

```diff
@@ -8,8 +8,6 @@
 
 import Foundation
 
-// note: This extension is only temporary, since we're
-// dropping iOS 11 in the upcoming major
```

Comment:
We already replaced the implementation with `ISO8601DateFormatter` and dropped iOS 11, and there's no need to remove this extensions since they're useful.

https://github.com/RevenueCat/purchases-hybrid-common/commit/dacb949cb7db73560fd057deea6026b727aa8717